wordpress-mobile/WordPress-Android
# Issue 12404/remove status and visibility option

Fixes #12404 

## Solution
All the logic that's related to the `Status & Visibility` functionality has been removed. 

From a code standpoint, verify that all functionality and related classes that are related to the `Status & Visibility` attribute has been removed. 

## Testing

1. Create a new post. 
2. Click Publish. 
3. Observe that the Status & Visibility option has been removed. 

<kbd><img src="https://user-images.githubusercontent.com/1509205/87080999-3bed3500-c1ee-11ea-81bc-7997044e8ab9.png" width="320"></kbd>
-------------------

1. Create a new page. 
2. Click Publish. 
3. Observe that the Status & Visibility option has been removed. 

<kbd><img src="https://user-images.githubusercontent.com/1509205/87081066-52938c00-c1ee-11ea-8515-939f00cbc283.png" width="320"></kbd>

## Concerns
There are times when the bottom sheet gets cut off due to issues with the navigation bar's height. This isn't related to this removal but will be noticed during testing so it's something that will be addressed. 

<kbd><img src="https://user-images.githubusercontent.com/1509205/87081278-ac945180-c1ee-11ea-9097-5d479c7337d3.gif" width="320"></kbd>

## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
